skbio/tree/_tree.py

```diff
@@ -3098,3 +3098,35 @@ def shuffle(self, k=None, names=None, shuffle_f=np.random.shuffle, n=1):
 
             yield self
             counter += 1
+
+    @experimental(as_of="0.5.2-dev")
+    def support(self):
+        """Return support value of a node if available
+
+        Returns
+        -------
+        float or None
+            support value of the node, or None if not available
+
+        Notes
+        -----
+        A "support value" is defined as the numeric form of a whole node label
+        without ":", or the part preceding the first ":" in the node label.
+        - For examples: "(a,b)1.0", "(a,b)1.0:2.5", and "(a,b)'1.0:species_A'".
```

Comment:
but an empty line before the examples; otherwise, they will be all concat into a paragraph. Also add a note to changelog file. Other than these, it looks good to me.

Reply:
Thank you very much! I modifed the code as you suggested. @RNAer